### PR TITLE
Hardsuit Loadout Colour not being saved fix

### PR DESCRIPTION
**Pull request by Wirpy**
## About The Pull Request

me being lazy and not PRing this (real). 
Coded by artur. 

This PR fixes the Hardsuits not having their secondary colour layer saved by the loadout vendor. 

## Why It's Good For The Game

This PR fixes the Hardsuits not having their secondary colour layer saved by the loadout vendor. 

## Changelog
:cl: Artur Lang
fix: Hardsuit secondary colours not being saved by loadout vendor
:cl:

### CODE REVIEW

**Comment by aarrisu** [comment]
WHAt!!!

**Review by TiviPlus — CHANGES_REQUESTED** [review]
https://discord.com/channels/498569646014857217/500801682058379265/1187316007619018813 pinged you

File: code/modules/clothing/modular_armor/attachments/secondary_color.dm [hunk 1]
```diff
@@ -34,20 +44,20 @@
 	current_variant = "normal"
 	icon_state_variants = list(
 		"normal",
-		"webbing",
+		"hardsuit webbing",
```
Comment (Arturlang) [inline code/modules/clothing/modular_armor/attachments/secondary_color.dm:47]: just undo this change and linters will be fixed

Comment (Wirpy) [reply code/modules/clothing/modular_armor/attachments/secondary_color.dm]: fixed

**Review by TiviPlus — APPROVED** [review]
